Userland/Libraries/LibGfx/ICCProfile.cpp

```diff
@@ -255,6 +271,22 @@ char const* color_space_name(ColorSpace color_space)
     }
 }
 
+char const* rendering_intent_name(RenderingIntent rendering_intent)
```

Comment:
Why no StringView?
(Is the NULL state ever supposed to be reached without the world burning around this function?)

Reply:
See discussion here: https://github.com/SerenityOS/serenity/pull/16724#discussion_r1059401594

(No.)